app/src/main/java/com/example/warehousemanagement/domain/model/Notification.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.warehousemanagement.domain.model
+
+import java.util.Date
+
+data class Notification(
+    val id: String,
+    val title: String,
+    val senderId: String,
+    val receiverId: String,
```

Comment:
Ủa gửi được cho đơn cá nhân hả

Reply:
Để tui chỉnh lại, mỗi lần thông báo là thông báo all

Reply:
Yub, this way is better for dev =)))